odoo/addons/base/tests/test_ormcache.py

```diff
@@ -116,7 +116,6 @@ def run(cache):
         )
 
     def test_signaling_01_single(self):
-        self.assertFalse(self.registry.test_cr)
```

Comment:
we could replace the assertion with something like `self.assertFalse(cls._registry_patched)`